CMakeLists.txt

```diff
@@ -72,41 +72,39 @@ option(CHRONOLOG_ENABLE_DOXYGEN "Enable Doxygen documentation generation." OFF)
 # TODO Should we consider the posibility of a user passing their own fflags through CMAKE_CXX_FLAGS
 #   Encapsulate everything around if(NOT CMAKE_CXX_FLAGS)
 #   Check Hermes for hwo they delete flags automatically
-if(SET_WERROR)
-    add_compile_options(-Wall)
-    add_compile_options(-Wextra)
-    add_compile_options(-Werror)
-else()
-    add_compile_options(-Wall)
-    add_compile_options(-Wextra)
-endif()
+#if(SET_WERROR)
+#    add_compile_options(-Wall)
+#    add_compile_options(-Wextra)
+#    add_compile_options(-Werror)
+#else()
+#    add_compile_options(-Wall)
+#    add_compile_options(-Wextra)
+#endif()
 
 # Change some compilation flags depending on the build type
 if(CMAKE_BUILD_TYPE STREQUAL "Debug")
     message("Compiling ChronoLog in Debug Mode")
 #    add_compile_options(-g3)
     add_compile_options(-O0)
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O0")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O0")
-
-    if (HERMES_USE_ADDRESS_SANITIZER)
-        add_compile_options(-fsanitize=address)
-    endif()
-
-    if (HERMES_USE_THREAD_SANITIZER)
-        add_compile_options(-fsanitize=thread)
-    endif()
-
-    if (HERMES_USE_THREAD_SANITIZER AND HERMES_USE_ADDRESS_SANITIZER)
-        message(FATAL_ERROR "Cannont use -fsanitize=address and -fsanitize=thread "
-                "at the same time")
-    endif()
-endif()
-
-if(CMAKE_BUILD_TYPE STREQUAL "Release")
+    #set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O0")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O0 -g")
+
+    #if (HERMES_USE_ADDRESS_SANITIZER)
+    #    add_compile_options(-fsanitize=address)
+    #endif()
+
+    #if (HERMES_USE_THREAD_SANITIZER)
+    #    add_compile_options(-fsanitize=thread)
+    #endif()
+
+    #if (HERMES_USE_THREAD_SANITIZER AND HERMES_USE_ADDRESS_SANITIZER)
+    #    message(FATAL_ERROR "Cannont use -fsanitize=address and -fsanitize=thread "
+    #            "at the same time")
+    #endif()
```

Comment:
Hermes is another project in our group. I think Jaime took the CMakeLists.txt from that project as the starting point.

Reply:
@JaimeCernuda Do you want to keep these options? I have commented them for now.